src/main/java/seedu/duke/model/project/Project.java

```diff
@@ -45,7 +45,7 @@ public Project(int projectID, String title, String description, int projectDurat
 
     public String toIdString() {
         StringBuilder projectInString = new StringBuilder();
-        projectInString.append(String.format("[Project ID: %s]", this.projectID));
+        projectInString.append(String.format("[Project ID: %s]\n", this.projectID));
```

Comment:
The code analysis stated this line don't need `StringBuilder` since you are only appending on string. you might wanna change that.
The newline should also use `%n` instead of `\n`. `%n` works similarly like the `System.lineSeparator()`.